src/common/dialog/AddonDispatchTsDialog.svelte

```diff
@@ -1,41 +1,28 @@
-<script lang="ts">
-  import Button from "@/common/Button";
-  import AddonRun from "@/common/AddonRun";
+<script>
```

Comment:
I could probably use some description as to why this file changed so much, it is a little hard to follow

Reply:
This one had a bunch of little things that were touched, plus one big formatting change. The big one was an outer `<div>` that wasn't doing anything, so I deleted it. That caused an indentation change on the whole component. But it also had a couple unused functions and imports, which I deleted, and then I added stuff, so there's a lot. This file just does a bunch of things.